### PR TITLE
#176 make past projects more dynamic

### PR DESCRIPTION
**Pull request by OttoChen312**
## Description

uncommented the past projects code in the projectjoin class
created a object in the project interface for pastprojects with a title and a description 
added the pastprojects property to each project object in projectwrapper
still a problem of the nova past project description strings not being able to be seperated into seperate lines

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by OttoChen312** [comment]
[Make past projects dynamic](https://trello.com/c/QRLvPd3R/176-make-past-projects-dynamic)

File: src/pages/Project/ProjectJoin.tsx [hunk 1]
```diff
@@ -147,25 +147,18 @@ const ProjectJoin: React.FC<ProjectProps> = (props) => {
       </section>
 
       {/* TODO: Make past projects dynamic */}
-      {/* <section className='full-bleed4'>
+       <section className='full-bleed4'>
                 <div className='past-positions-and-projects' style={{ paddingBottom: "2%", paddingTop: "3%" }}>
                     <p style={{ paddingTop: "3%" }}>03</p>
                     <h2>{TEXT.PROJECT_JOIN.PAST_POSITIONS_AND_PROJECTS.TITLE}</h2>
+                    {props.project.pastProjects?.map(items =>
```
Comment (ryanyae) [inline src/pages/Project/ProjectJoin.tsx:154]: Just one request, can you just add padding to the individual past projects? 
image

**Review by ryanyae — CHANGES_REQUESTED** [review]
Thank you Otto! All this looks great!

**Comment by ElisaHu1** [comment]
also there is a conflict file @OttoChen312 if you could modify it

File: src/pages/Project/ProjectJoin.tsx (hunk 1 above)
Comment (ryanyae) [reply src/pages/Project/ProjectJoin.tsx:154]: Do you think you could also adding padding to the left and the right?

**Review by ryanyae — CHANGES_REQUESTED** [review]
Just resolve the conflicts and I will be able to merge this into main!

**Review by ElisaHu1 — APPROVED** [review]
looks great!

**Review by ryanyae — APPROVED** [review]
Sorry for keep this PR blocked but great job Otto! Thank you for your hard work!

**Pull request merged**